### PR TITLE
Documentation on running ML/ MT RL experiments

### PR DESCRIPTION
**Pull request by irisliucy**
This PR is on this #1542

### CODE REVIEW

**Comment by irisliucy** [comment]
@Mergifyio rebase

File: docs/user/meta_multi_task_rl_exp.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+- `RL2Env`: a specific wrapper for RL2 environment
+
+Also, it's worth noting that [Meta-World](https://github.com/rlworkgroup/metaworld) provides a variety of benchmarked robotics tasks and can be extensively used in garage.
+
+## Meta-RL experiments
+
+The garage repository contains several meta-RL experiment examples. We will take a look at `te_ppo_metaworld_ml1_push.py` as below:
+
+```py
+from metaworld.benchmarks import ML1
```
Comment (haydenshively) [inline docs/user/meta_multi_task_rl_exp.md:24]: @krzentner @avnishn Do examples like this need to be changed to work with the new MetaWorld API?

Comment (krzentner) [reply docs/user/meta_multi_task_rl_exp.md:24]: We haven't written the necessary `TaskSampler` yet, so they really can't.

Comment (krzentner) [reply docs/user/meta_multi_task_rl_exp.md:24]: Let's just merge this for now, with the understanding that we still have a bunch of (re)writing to do once we get support for the new MetaWorld API in garage.

File: docs/user/meta_multi_task_rl_exp.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+
+```
+
+In this example, to sample tasks from `MT50` environments in a round robin fashion, the call to `MultiEnvWrapper` becomes `MultiEnvWrapper(envs , sample_strategy=round_robin_strategy, mode='del-onehot')`.
+
+## Garage's meta-/ multi-task RL benchmark
+
+Garage benchmarks the following meta-/ multi RL experiements:
+
+```eval_rst
```
Comment (gitanshu) [inline docs/user/meta_multi_task_rl_exp.md:292]: This doesn't seem to be rendering properly
https://garage--1802.org.readthedocs.build/en/1802/user/meta_multi_task_rl_exp.html

File: docs/user/meta_multi_task_rl_exp.md [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+| Algorithm     | Observation | Action   | Environment Set                                                            |
+|---------------|-------------|----------|----------------------------------------------------------------------------|
+| Meta-RL       | Non-Pixel   | Discrete | 'ML1-push-v1' ,  'ML1-reach-v1' ,  'ML1-pick-place-v1' ,  'ML10' ,  'ML45' |
+| Multi-Task RL | Non-Pixel   | Discrete | 'ML1-push-v1' ,  'ML1-reach-v1' ,  'ML1-pick-place-v1' ,  'MT10' ,  'MT50' |
+|               |             |          |                                                                            |
+```
+
+See `docs/benchmarking.md` for a more detailed explaination on garage's benchmarking.
```
Comment (gitanshu) [inline docs/user/meta_multi_task_rl_exp.md:300]: I think this should link to the document being referred. A file system path in documentation is not useful.

Comment (irisliucy) [reply docs/user/meta_multi_task_rl_exp.md:300]: You're right. Added the doc link.

**Comment by irisliucy** [comment]
@Mergifyio rebase

File: docs/user/meta_multi_task_rl_exp.md [hunk 4]
```diff
@@ -0,0 +1,313 @@
+# Run Meta-/Multi-Task RL Experiments
+
+This guide will walk you through how to run meta-/ multi-task RL experiments in garage.
+
+Similar to running all other experiments in garage, running meta-/ multi-task RL experiments generally involves steps such as:
+
+- defining the experiement with the `wrap_experiment` decorator;
```
Comment (ryanjulian) [inline docs/user/meta_multi_task_rl_exp.md:7]:
```suggestion
- Defining the experiement with the `wrap_experiment` decorator;
```

File: docs/user/meta_multi_task_rl_exp.md [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+- constructing a `LocalRunner`;
```
Comment (ryanjulian) [inline docs/user/meta_multi_task_rl_exp.md:8]:
```suggestion
- Constructing a `LocalRunner`
```

File: docs/user/meta_multi_task_rl_exp.md [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
+- constructing an environment;
```
Comment (ryanjulian) [inline docs/user/meta_multi_task_rl_exp.md:9]:
```suggestion
- Constructing an environment
```

File: docs/user/meta_multi_task_rl_exp.md [hunk 7]
```diff
@@ -0,0 +1,313 @@
...
+- constructing policy/ agorithm object(s).
```
Comment (ryanjulian) [inline docs/user/meta_multi_task_rl_exp.md:10]:
```suggestion
- Constructing policy and algorithm objects
```

File: docs/user/meta_multi_task_rl_exp.md [hunk 8]
```diff
@@ -0,0 +1,313 @@
...
+
+In meta-/ multi-task RL experiment, it revolves around solving multiple tasks and hence the construction of multiple / specific environment(s). Belows are a few environment wrappers commonly used in garage for meta-/ multi-task RL learning:
```
Comment (ryanjulian) [inline docs/user/meta_multi_task_rl_exp.md:12]:
```suggestion
In meta-/multi-task RL experiment, it revolves around solving multiple tasks and hence the construction of multiple/specific environment(s). Belows are a few environment wrappers commonly used in garage for meta-/multi-task RL learning:
```

File: docs/user/meta_multi_task_rl_exp.md [hunk 9]
```diff
@@ -0,0 +1,313 @@
...
+
+- `MultiEnvWrapper`: a wrapper for mulitiple environments
+- `RL2Env`: a specific wrapper for RL2 environment
+
+Also, it's worth noting that [Meta-World](https://github.com/rlworkgroup/metaworld) provides a variety of benchmarked robotics tasks and can be extensively used in garage.
```
Comment (ryanjulian) [inline docs/user/meta_multi_task_rl_exp.md:17]:
```suggestion
Also, it's worth noting that [MetaWorld](https://github.com/rlworkgroup/metaworld) provides a variety of benchmarked robotics tasks and can be extensively used in garage.
```

**Pull request merged**